charts/minecraft/values.yaml

```diff
@@ -171,7 +171,7 @@ minecraftServer:
   eula: "FALSE"
   # One of: LATEST, SNAPSHOT, or a specific version (ie: "1.7.9").
   version: "LATEST"
-  # This can be one of "VANILLA", "FORGE", "SPIGOT", "BUKKIT", "PAPER", "FTBA", "SPONGEVANILLA", "CURSEFORGE"
+  # This can be one of "VANILLA", "FORGE", "SPIGOT", "BUKKIT", "PAPER", "FTBA", "SPONGEVANILLA", "AUTO_CURSEFORGE"
```

Comment:
Perhaps the rather than list them off exhaustively a second line of comment could point users to https://docker-minecraft-server.readthedocs.io/en/latest/types-and-platforms/ for the full set of options. For example `MODRINTH` also would need to be listed.

Reply:
I just added the change because i was irritated by using the CURSEFORGE/AUTO_CURSEFORGE.
I also added the link.
Should i update the chart version?
Feel free to commit on the branch.

Reply:
Yep, that's totally fair

Please change the link to be the one I provided. The one you used is to a specific sub-section.

Yes, chart version still needs to be bumped. Sorry...I inherited the build setup as it is.

Reply:
np. i like strict versioning :) 
Now it should be right?